concent_api/core/message_handlers.py

```diff
@@ -75,8 +75,8 @@ def handle_send_force_report_computed_task(client_message):
         requestor_public_key,
         task_to_compute,
     )
-
-    if Subtask.objects.filter(
+    # QuerySet is inferred as list; pylint: disable=no-member; https://github.com/PyCQA/pylint-django/issues/165
+    if Subtask.objects.filter(  # pylint: disable=no-member
```

Comment:
Please reword it:

``` python
# We have to disable no-member warning on Subtask queries because pylint thinks the result is a list, not QuerySet.
# See this pylint bug: https://github.com/PyCQA/pylint-django/issues/165
```